File: Codewars/7kyu/7kyu_mumbling/src/other_1.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	fmt.Println(Accum("abcd") == "A-Bb-Ccc-Dddd")
	fmt.Println(Accum("RqaEzty") == "R-Qq-Aaa-Eeee-Zzzzz-Tttttt-Yyyyyyy")
	fmt.Println(Accum("cwAt") == "C-Ww-Aaa-Tttt")
}

func Accum(s string) string {
	words := make([]string, len(s))

	for i, c := range s {
		words[i] = strings.Title(strings.Repeat(strings.ToLower(string(c)), i+1))	
	}

	return strings.Join(words, "-")
}

File: Codewars/7kyu/7kyu_cartesian_neighbors/src/other_1.py
def cartesian_neighbor(x, y):
    return [(x+i, y+j) for i in range(-1, 2) for j in range(-1, 2) if i or j]

print [(1, 1), (1, 2), (1, 3), (2, 1), (2, 3), (3, 1), (3, 2), (3, 3)] == cartesian_neighbor(2, 2)
